### DISCUSSION TITLE
https://greendolphindance.com/2024/05/15/Category%E9%A1%B5%E9%9D%A2%E8%BF%87%E9%95%BF%E5%8D%9A%E6%96%87%E5%90%8D%E7%9C%81%E7%95%A5/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#26] [General]
# https://greendolphindance.com/2024/05/15/Category%E9%A1%B5%E9%9D%A2%E8%BF%87%E9%95%BF%E5%8D%9A%E6%96%87%E5%90%8D%E7%9C%81%E7%95%A5/

Back to the rough ground!

https://greendolphindance.com/2024/05/15/Category%E9%A1%B5%E9%9D%A2%E8%BF%87%E9%95%BF%E5%8D%9A%E6%96%87%E5%90%8D%E7%9C%81%E7%95%A5/

**Comment by ouuan** [comment]
感觉正确的修复方式似乎是给 `archive-item-link` 设置 `max-width: 100%` 🤔️ F12 选择 `archive-item-link` 可以观察到它的宽度会超过 parent

**Reply by greendolphindance** [reply]
我回头研究一下！
